testnet/shared/config.yaml

```
@@ -11,9 +11,10 @@ CONFIG_NAME: 'lukso'
 # ---------------------------------------------------------------
 # Min Genesis Validators in Genesis state
 MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 4000
-# Date and time (GMT): Monday, 2 May 2023 15:45:00
-MIN_GENESIS_TIME: 1683042356
-# 2400 seconds (40 Min) / 16128 + 1683042356 + 2400 - 2
+# Date and time (UTC): Wednesday, 3 May 2023 16:20:00
+# Shanghai time = 16128 (42 epochs in seconds) + 1683128400 + 2400 - 2
+MIN_GENESIS_TIME: 1683128400
```

Is it worth using `_` as thousand separators for readability? 
I would just suggest this unless this is not the convention for writing epoch and timestamps (easy to write a number incorrectly without noticing it otherwise).

```suggestion
MIN_GENESIS_TIME: 1_683_128_400
```